tests/conftest.py

```diff
@@ -58,6 +59,11 @@
 
 
 METRICS = get_metrics_logger()
+QEMU_PUB_KEY_URL = "https://keys.openpgp.org/vks/v1/by-fingerprint/CEACC9E15534EBABB82D3FA03353C9CEF108B584"
```

Comment:
The Ubuntu packages don't have the vhost-user-blk binary?

Reply:
that was my first guess too, but I wasn't able to find it.